osu.Game/Overlays/Direct/PlayButton.cs

```diff
@@ -18,6 +18,7 @@ namespace osu.Game.Overlays.Direct
 {
     public class PlayButton : Container
     {
+        private static PlayButton activeBeatmapPreview;
```

Comment:
Hmm I don't really like this static. Here's a suggestion:

Create a new audio collection manager, say, `PreviewTrackManager`, instantiate it in `OsuGameBase` add it as an item to the global `AudioManager`, dependency cache it. Some of this is already done in `TrackLoader`, as an example, but you're going to have to change things up a bit - probably have it be a drawable with an encapsulated collection.

Have it handle most of what's done here in `PlayButton`:

* The volume stuff done in `TrackLoader`.
* The muting of audio track.
* The retrieval from the online store.

Essentially what I'd really want is for `PlayButton` to be just:

```
PreviewTrack track;

load(PreviewTrackManager previewTracks)
{
    track = previewTracks.Get("http://my.preview...");
}

void play() => track.Start();
void stop() => track.Stop();